extensions/sql-migration/src/wizard/assessmentDetailsPage/treeComponent.ts

```diff
@@ -59,7 +59,7 @@ export class TreeComponent {
 	private _disposables: vscode.Disposable[] = [];
 	private _model!: MigrationStateModel;
 
-	constructor(public wizard: azdata.window.Wizard) { }
+	constructor(public wizard: azdata.window.Wizard, private _readOnly: boolean = false) { }
```

Comment:
Should the first variable be private as well?

Reply:
Here wizard is intentionally made public, so that changes made by other classes while this class is being executed is reflected. Public access modifier makes the code accessible for all classes.


<!--EndFragment-->